src/routes/docs/tutorials/sveltekit-ssr-auth/step-8/+page.markdoc

```diff
@@ -0,0 +1,11 @@
+---
+layout: tutorial
+title: All set
+description: Add Authentication to a SvelteKit project using Appwrite.
+step: 8
+---
+If you want to see the complete source code, see the [demos-for-svelte](https://github.com/appwrite/demos-for-svelte/tree/main/server-side-rendering) repository.
+
+## Other authentication methods
```

Comment:
Please use `#` instead of `##` because internal logic demotes tags by one level.

Also check for tags where you don't have {% #link-to-header %}

Reply:
Done